wp-content/themes/eighshift-task/src/rest/fields/class-posts-fields.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * REST Field on /posts/ route
+ *
+ * @since 1.0.0
+ * @package Eighshift_Task\Rest\Fields
+ */
+
+namespace Eighshift_Task\Rest\Fields;
+
+use Eightshift_Libs\Core\Service;
+use Eightshift_Libs\Routes\Registrable_Field;
+
+/**
+ * Class for registering and changine REST fields for /posts/ route.
+ */
+class Posts_Fields implements Service, Registrable_Field {
+
+  /**
+   * Hook registration
+   *
+   * @since 1.0.0
+   */
+  public function register() : void {
+    add_action( 'rest_api_init', [ $this, 'register_field' ] );
+  }
+
+  /**
+   * Register fields
+   */
+  public function register_field() : void {
+    register_rest_field(
+      'post',
+      'featuredImage',
+      [
+        'get_callback' => [ $this, 'featured_image_callback' ],
+      ]
+    );
+
+    register_rest_field(
```

Comment:
nuop, every field its own file not like this

Reply:
👍

Reply:
👍